docs/charon/charon-cli-reference.md

```diff
@@ -101,22 +89,23 @@ Usage:
   charon create cluster [flags]
 
 Flags:
-      --clean                          Delete the cluster directory before generating it.
-      --cluster-dir string             The target folder to create the cluster in. (default ".charon/cluster")
-      --definition-file string         Optional path to a cluster definition file or an HTTP URL. This overrides all other configuration flags.
-      --fee-recipient-address string   Optional Ethereum address of the fee recipient
-  -h, --help                           Help for cluster
-      --insecure-keys                  Generates insecure keystore files. This should never be used. It is not supported on mainnet.
-      --keymanager-addresses strings   Comma separated list of keymanager URLs to import validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being imported to the first address, node1's keyshares to the second, and so on.
-      --name string                    The cluster name
-      --network string                 Ethereum network to create validators for. Options: mainnet, gnosis, goerli, kiln, ropsten, sepolia. (default "goerli")
-      --nodes int                      The number of charon nodes in the cluster. Minimum is 4. (default 4)
-      --num-validators int             The number of distributed validators needed in the cluster. (default 1)
-      --split-existing-keys            Split an existing validator's private key into a set of distributed validator private key shares. Does not re-create deposit data for this key.
-      --split-keys-dir string          Directory containing keys to split. Expects keys in keystore-*.json and passwords in keystore-*.txt. Requires --split-existing-keys.
-      --threshold int                  Optional override of threshold required for signature reconstruction. Defaults to ceil(n*2/3) if zero. Warning, non-default values decrease security.
-      --withdrawal-address string      Ethereum address to receive the returned stake and accrued rewards. (default "0x0000000000000000000000000000000000000000")
```

Comment:
I don't think this is a breaking change. The new `--withdrawal-addresses` flag says:
`
--withdrawal-addresses strings Comma separated list of Ethereum addresses to receive the returned stake and accrued rewards for each validator. Either provide a single withdrawal address or withdrawal addresses for each validator.
`

You can provide a *single* withdrawal address *OR* multiple withdrawal addresses.

Reply:
Yes but for example, in our guides https://docs.obol.tech/docs/next/int/quickstart/quickstart-alone#step-2-create-the-artifacts-needed-to-run-a-dv-cluster this command won't work anymore, because `--withdrawal-address` is not a recognised flag anymore.

Reply:
Sweet. If the two places you made the change are the only places where it shows up, I'm probably happy to merge.